src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/jalangi/JalangiAdapter.java

```diff
@@ -150,6 +150,7 @@ private Object getConfig() {
     }
 
     @ExportMessage
+    @TruffleBoundary
```

Comment:
I was wondering how this boundary would affect performance, i.e. whether it would be useful for at least inlining to be allowed into the API calls. I'll create a new branch with a possible optimization for that.

Reply:
This should be ok as it would only affect the uses of `J$.xxx`.

Reply:
A few `J$` methods are called often though, but I'm not sure whether they would be ever inlined by the JIT.  Follow-up in !91